dotnet-desktop-guide/winforms/migration/snippets/clipboard-dataobject-net10/net/csharp/PrimitiveTypesExamples.cs

```diff
@@ -0,0 +1,66 @@
+using System;
+using System.Collections.Generic;
+using System.Windows.Forms;
+
+namespace ClipboardExamples
+{
+    public class PrimitiveTypesExamples
+    {
+        // <PrimitiveTypesExample>
+        public static void PrimitiveTypesExample()
+        {
+            // Numeric types
+            Clipboard.SetData("MyInt", 42);
+            Clipboard.SetData("MyDouble", 3.14159);
+            Clipboard.SetData("MyDecimal", 123.45m);
+
+            // Text and character types
+            Clipboard.SetData("MyString", "Hello World");
+            Clipboard.SetData("MyChar", 'A');
+
+            // Boolean and date/time types
+            Clipboard.SetData("MyBool", true);
+            Clipboard.SetData("MyDateTime", DateTime.Now);
+            Clipboard.SetData("MyTimeSpan", TimeSpan.FromMinutes(30));
+
+            // Later retrieval with type safety
+            if (Clipboard.TryGetData("MyInt", out int value))
```

Comment:
This won't work. The clipboard only has a single item, which would be `"MyTimeSpan"` in this case.